src/gateways/MsSQL/retrieveEmailAndHashedPassword.js

```diff
@@ -0,0 +1,49 @@
+import logger from "../../../logger";
```

Comment:
Could you add a unit test for this gateway please?

Reply:
Unit test added